github/ci/prow-sig-scale-workloads/roles/bootstrap/tasks/main.yml

```diff
@@ -0,0 +1,133 @@
+---
+- name: disable swap
+  block:
+    - name: swapoff
+      shell: |
+        swapoff -a
+    - name: comment swap entry from fstab
+      replace:
+        path: /etc/fstab
+        regexp: '(.*swap.*)'
+        replace: '#\1'
+    - name: mask systemctl swap unit
+      systemd:
+        name: dev-sda3.swap
+        masked: yes
+
+- name: enable mirrors for yum repos
+  block:
+    - name: find repo files
+      find:
+        paths: '/etc/yum.repos.d'
+        patterns: 'CentOS-Linux-*'
+      register: repo_entries
+      changed_when: False
+
+    - name: uncomment mirrorlist lines
+      lineinfile:
+        path: '{{ item.path }}'
+        regexp: '#(\s?mirrorlist.*)'
+        line: '\1'
+        backrefs: yes
+      with_items: '{{ repo_entries.files }}'
+
+    - name: comment baseurl line
+      lineinfile:
+        path: '{{ item.path }}'
+        regexp: '^baseurl(.*)'
+        line: '# baseurl\1'
+        backrefs: yes
+      with_items: '{{ repo_entries.files }}'
+
+- name: remove uneeded packages
+  yum:
+    name:
+      - podman
+      - buildah
+      - runc
+      - containerd
+      - containerd.io
+      - crio
+      - docker
+      - docker-client
+      - docker-client-latest
+      - docker-common
+      - docker-latest
+      - docker-latest-logrotate
+      - docker-logrotate
+      - docker-engine
+      - docker-ce 
+      - docker-ce-cli 
+    state: absent
+
+- name: NetworkManager DNS config
+  lineinfile:
+    path: /etc/sysconfig/network-scripts/ifcfg-bond1
```

Comment:
Same here, have you checked if the nodes have a `bond1` nic?

Reply:
Yes, `bond0` and `bond1`